common/app/views/fragments/collections/popular.scala.html

```diff
@@ -11,38 +11,72 @@
         @if(isTabbed) {
```

Comment:
I'd add a custom data-link-name to line 10 here so clicks on this can be split out in Omniture. 

Just something like data-link-name="most popular @if(isMostPopularDefaultTest){ | ab-most-popular-default-test}"